Fix build for non-VSX altivec powerpc CPUs.

he32 opened the pull request:
Fixes https://github.com/randombit/botan/issues/3958

randombit commented:
LGTM but the formatter check is complaining. You should just be able to apply the diff printed in the failing job.

he32 commented:
> LGTM but the formatter check is complaining. You should just be able to apply the diff printed in the failing job.

I think I fixed the formatting and squashed the two commits back into one.

he32 commented:
And ... made another formatting fix and squash.
